src/components/application_manager/include/application_manager/commands/mobile/set_app_icon_request.h

```diff
@@ -89,7 +89,8 @@ class SetAppIconRequest : public CommandRequestImpl {
    * @param icons_amount Amount of icons to be deleted
    */
   void RemoveOldestIcons(const std::string& storage,
-                         const uint32_t icons_amount) const;
+                         const uint32_t icons_amount,
+                         bool& is_correct_removed) const;
```

Comment:
@VVeremjova, @tkireva I agree, this parameter is "out parameter" so it should indicate this by "out_" prefix, also you should add this field to doxygen of  "RemoveOldestIcons" method.